File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/MealOptionViewModel.cs
using System.Collections.Generic;

namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class MealOptionViewModel : BaseViewModel
    {
        public MealOptionViewModel()
        {
            this.CuisineOptions = new List<CuisineTypeViewModel>();
        }
        public int? CurrencyId { get; set; }
        public double Price { get; set; }
        public List<CuisineTypeViewModel> CuisineOptions { get; }
    }
}

File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/associatemealplanviewmodel/mealoptionviewmodel.ts
import { CuisineViewModel } from './cuisineviewmodel';
import { IObjectWithState } from '../../../common/iobjectwithstate';
import { ObjectState } from '../../../common/enums';

export class MealOptionViewModel implements IObjectWithState {
    objectState: ObjectState;
    currencyId: number;
    price: number;
    cuisineOptions: CuisineViewModel[];
}
